telemetry/packages/public-app/components/accelerationChart.tsx

```diff
@@ -0,0 +1,52 @@
+'use client';
+
+import { Card, LineChart, Title } from '@tremor/react';
+import { useQuery } from 'react-query';
+import format from 'date-fns/format';
+
+const valueFormatter = (number: any) =>
+  `$ ${new Intl.NumberFormat('us').format(number).toString()}`;
+
+export const AccelerationChart = () => {
+  const { data, isLoading, isError } = useQuery(
+    'displacement    ',
+    async () =>
+      (await fetch(
+        `${process.env.NEXT_PUBLIC_TELEMETRY_SERVER}/pods/pod_1/public-data/displacement?start=0`,
+      ).then(res => res.json())) as {
+        id: 'displacement';
+        timestamp: string;
```

Comment:
This would look a lot neater if it was extracted out, perhaps into a custom hook. I can look at this with you on Thursday